lib/matplotlib/backends/web_backend/mpl.js

```diff
@@ -125,6 +125,23 @@ mpl.figure.prototype._init_canvas = function() {
     // Throttle sequential mouse events to 1 every 20ms.
     rubberband.mousemove('motion_notify', mouse_event_fn);
 
+    function scroll_fn(event) {
+        event['data'] = 'scroll'
+        // http://www.adomas.org/javascript-mouse-wheel/
+        if (event.wheelDelta) { /* IE/Opera. */
+                event.step = event.wheelDelta/120;
+        } else if (event.detail) { /** Mozilla case. */
+            /** In Mozilla, sign of delta is different than in IE.
+             * Also, delta is multiple of 3.
+             */
+            event.step = -event.detail/3;
+        }
+        mouse_event_fn(event);
+    }
+
+    $(canvas_div)[0].addEventListener('DOMMouseScroll', scroll_fn, false);
```

Comment:
Can you just do `canvas_div.addEventListener(...)` here?  We have (what I think) is a reference to the js version of the div so there is no need to look it up.